From 0c2f6522ec6674423301748907c1897c0715a522 Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Mon, 7 Jan 2019 00:23:59 -0600
Subject: [PATCH] Test BLAS and LAPACKE in addition to LAPACK

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1ee66f1ad..74f4857cd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -358,6 +358,8 @@ endif()
 
 if(BUILD_TESTING)
   add_subdirectory(TESTING)
+  add_subdirectory(BLAS/TESTING)
+  add_subdirectory(LAPACKE/example)
 endif()
 
 # --------------------------------------------------
-- 
2.38.1.windows.1

