From bf00545ca6e745d1d59ff28c5bbdd63d1ec16ace Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Mon, 7 Jan 2019 00:23:59 -0600
Subject: [PATCH 1/6] Test BLAS and LAPACKE in addition to LAPACK

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d9dbf9746..dabab4c02 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -295,6 +295,8 @@ endif()
 
 if(BUILD_TESTING)
   add_subdirectory(TESTING)
+  add_subdirectory(BLAS/TESTING)
+  add_subdirectory(LAPACKE/example)
 endif()
 
 # --------------------------------------------------
