From 79ca675185264061e46274d5404e67bc937f04c6 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sun, 27 Oct 2024 00:15:33 +1100
Subject: [PATCH 4/6] use __declspec(selectany) for cblas_xerbla's RowMajorStrg
 on windows

---
 CBLAS/testing/c_xerbla.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CBLAS/testing/c_xerbla.c b/CBLAS/testing/c_xerbla.c
index c3f14c364..313c2a0e3 100644
--- a/CBLAS/testing/c_xerbla.c
+++ b/CBLAS/testing/c_xerbla.c
@@ -9,7 +9,7 @@ void cblas_xerbla(CBLAS_INT info, const char *rout, const char *form, ...)
 {
    extern int cblas_lerr, cblas_info, cblas_ok;
    extern int link_xerbla;
-   extern int RowMajorStrg;
+   extern __declspec(selectany) int RowMajorStrg;
    extern char *cblas_rout;
 
    /* Initially, c__3chke may call this routine with
